docs/usage.md

```diff
@@ -4,60 +4,306 @@
+
+- `"LT04"`: Landsat 4 TM
+- `"LT05"`: Landsat 5 TM
+- `"LE07"`: Landsat 7 ETM+
+- `"LC08"`: Landsat 8 OLI
+- `"S2A"`: Sentinel-2A MSI
+- `"S2B"`: Sentinel-2B MSI
+
+The second parameter - `sensors_level2` - controls the selection of satellites for the higher level processing steps. The parameter has to follow the FORCE notation for level 2 processing. In particular, a string containing space-separated satellite identifiers has to be supplied (e.g. `"LND04 LND05"` to include Landsat 4 and 5). Note that these identifiers differ from those used for the `sensors_level1` parameter.
+More details on available satellite identifiers can be found [here](https://force-eo.readthedocs.io/en/latest/components/higher-level/tsa/param.html), some common options include:
+
+- `"LND04"`: 6-band Landsat 4 TM
+- `"LND05"`: 6-band Landsat 5 TM
+- `"LND07"`: 6-band Landsat 7 ETM+
+- `"LND08/09"`: 6-band Landsat 8-9 OLI
+- `"SEN2A"`: 10-band Sentinel-2A
+- `"SEN2B"`: 10-band Sentinel-2B
+
+Note that the identifiers specified for both processing levels have to match the data made available to the workflow. In other words, satellite data for e.g. Landsat 5 can't be processed if it was not supplied using the `input` parameter.
+
+Both parameters can be passed as using:
+
+```bash
+--sensors_level1 = '[preprocessing satellite identifier string]'
+--sensors_level2 = '[higher level processing satellite identifier string]'
+```
+
+Note that both parameters are optional and are by default set to: `"LT04,LT05,LE07,S2A"` and `"LND04 LND05 LND07"`. Therefore, by default, the pipeline will use Landsat 4,5,7, and Sentinel 2 for preprocessing, while using Landsat 4,5 and 7 for higher level processing.
+
+### Resolution
+
+Resolution of satellite imagery defines the real size of a single pixel. As an example, a resolution of 30 meters indicates that a single pixel in the data covers a 30x30 meters square of the earths surface. Users can customize the resolution that FORCE should assume. This does not necessarily have to match the resolution of the supplied data. FORCE will treat imagery as having the specified resolution. However, passing a resolution not matching the satellite data might lead to unexpected results. Resolution is specified in meters.
+
+A custom resolution can be passed using:
+
+```bash
+--resolution '[integer]'
+```
+
+The default value is 30, as most Landsat satellite natively provide this resolution.
+
+### Temporal extent
+
+In some scenarios, user may be interested to limit the temporal extent of analysis. To enables this, users can specify both start and end date in a string with this syntax: `'YYYY-MM-DD'`.
+
+Start and end date can be passed using:
+
+```bash
+--start_date '[YYYY-MM-DD]'
+--end_date   '[YYYY-MM-DD]'
+```
+
+Default values are `'1984-01-01'` for the start date and `'2006-12-31'` for the end date.
```

Comment:
```suggestion
```
we show the default values on the parameters page, so easier to keep the docs in sync, by only having them in one place (if no further explanation of the choice of default values is given)